### DISCUSSION TITLE
Windows setup instructions

### DISCUSSION THREAD

**Discussion by CodeAbuser** [NginxProxyManager/nginx-proxy-manager#1849] [Q&A] [answered]
I don't understand how to set this up using Docker installed on Windows.  Where do I put the docker-compose.yml file? Where do I run the "docker-compose up -d" command?

Thanks for any help.

**Comment by CodeAbuser** [comment, marked as answer]
It appears that you just need to do the following which is not clear in the instructions:

1. create a folder on your drive (I created one at c:\tools\nginx-proxy-manager
2. go into that folder and create a file called docker-compose.yml and paste the contents from the setup instructions into that file and save it
3. open a command prompt and navigate to the folder you created and then run the 'docker-compose up -d' command.

**Reply by slaszt** [reply]
Thank you so much, been stuck on step 1 of the guide for literal months. Meanwhile my server has been unsecured this whole time because the instructions are made for developers and I'm just a person who has a PC.
